packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -25,9 +31,46 @@ export default defineComponent<KottiButton.PropsInternal>({
 	name: 'KtButton',
 	props: makeProps(KottiButton.propsSchema),
 	setup(props, { emit, slots }) {
+		const contentRef = ref<Element | null>(null)
+		const triggerRef = ref<Element | null>(null)
+
 		const hasSlot = computed(() => Boolean(slots.default))
 
+		const isIconButton = computed(
+			() => props.icon !== null && props.label === null,
+		)
+
+		const showHelpText = computed(
+			() => isIconButton.value && props.helpText !== null,
+		)
+
+		useTippy(
```

Comment:
Instead of setting the trigger to manual (lines 58 to 60), I would do:
```suggestion
		if (showHelpText.value)
			useTippy(
```
This way, Tippy isn't set needlessly for buttons without help text.
And lines 58 to 60 will not longer be needed.

Reply:
I'm not sure this condition would retrigger if the showHelpText changes during the lifecycle of the component though @santiagoballadares - therefore if it (helpText)'s set to null on mounting then gets a value from the user-app side, this might not work at all

hooks shouldn't be wrapped in if conditions on setup level as far as I recall from best practises guidelines as well

Reply:
Agreed @carsoli , didn't consider that this would change after mount